docs/section10/10-1.md

```diff
@@ -1,15 +1,15 @@
 # Airside systems – Rule 10-1  
-**Schema Version:** 0.0.23
-**Mandatory Rule:** False    
-**Rule ID:** 10-1  
+**Schema Version:** 0.0.36  
+**Mandatory Rule:** False     
+**Rule ID:** 10-1   
  
 **Rule Description:** The proposed design includes humidification and the baseline building design has been modeled with adiabatic humidification if proposed design does not comply with 90.1-2019 Section 6.5.2.4 and non-adiabatic humidification otherwise. 
 
```

Comment:
the rule description is slightly confusing. Maybe replace it with this?

When the proposed design includes humidification and complies with Section 6.5.2.4, then the baseline building design shall use adiabatic humidification.
When the proposed design includes humidification and does not comply with Section 6.5.2.4 then the baseline building design shall use nonadiabatic humidification.

Reply:
@supriyagoel I agree. I made the update but modified it as your text appears to flip the adiabatic and nonadiabatic. Thanks!